README.md

```diff
@@ -125,31 +125,31 @@ PDF files are also available from [here](https://github.com/euslisp/jskeus/raw/m
 
 #### 3.6.1 Installing dependent libraries
 
-for Ubuntu users
+For Ubuntu users:
 ```
 $ sudo apt-get install git make gcc g++ libjpeg-dev libxext-dev libx11-dev libgl1-mesa-dev libglu1-mesa-dev libpq-dev libpng12-dev xfonts-100dpi xfonts-75dpi gsfonts-x11 texlive-fonts-extra xfonts-100dpi-transcoded xfonts-75dpi-transcoded msttcorefonts
 ```
 
-for Mac OSX users using Homebrew
+For Mac OSX users using Homebrew:
 ```
 $ brew install jpeg libpng mesalib-glw wget
 ```
 
 '''NOTE:'''
-EusLisp needs X11 Library, but Mac OSX mountain lion no longer support it. So you need to install substitution [XQuartz](http://xquartz.macosforge.org/landing/).
+EusLisp needs X11 Library, but Mac OSX mountain lion no longer supports it. Given this case, you'll need to install [XQuartz](http://xquartz.macosforge.org/landing/).
 
-Also for Mac OSX lion (or older) users it's better to install the latest XQuartz for the safety.
+Also for Mac OSX lion (or older) users it's better to install the latest XQuartz for safety and compatibility reasons.
 
 (If XQuartz is older than 2.7.3, the installation of EusLisp will fail. You should install the latest XQuartz.)
 
 
-for Raspberry Pi users using debian(Raspbian "wheezy")
+For Raspberry Pi users using debian(Raspbian "wheezy"):
 ```
 $ sudo apt-get install git gcc g++ libjpeg62-dev libxext-dev libx11-dev libgl1-mesa-dev libglu1-mesa-dev libpq-dev libpng12-dev xfonts-100dpi xfonts-75dpi
 ## additional install for using OpenGL on raspberry pi
 $ sudo apt-get install libgl1-mesa-dri xserver-xorg-video-vesa xutils
 ```
-for Cygwin(now working with 32bit only)
+For Cygwin(doesn't work with 32 bit architectures):
```

Comment:
@oxcabe It looks you are changing to the opposite.

Reply:
Whoops. I think i read "not" instead of "now".